### PR TITLE
Created Vertical Navbar Component

### LINKED ISSUE #16: 14 - Dashboard Side Navbar
**Description**
Create the Side Navbar for the User/Admin Dashboard pages.

**Acceptance Criteria**
Code is clean and properly formatted
Component reflects Figma design (does not have to exactly match, simplicity > perfection)

Steps:

- Create the navbar component under the components folder named vertical_navbar.tsx
- Ensure that component matches navbar in [Figma ](https://www.figma.com/file/Mv4wKS0WTiszMuiDbLcpTi/General-VMS-Figma?type=design&node-id=86%3A218&mode=design&t=XGONHxjuloBOV8pF-1)(User Dashboard)

**Additional Information**
image

### PR DESCRIPTION
**Pull request by soramicha**
## Developer: Sophia Chang

Closes #25

### Pull Request Summary

Created a vertical navbar component using TailwindCSS.

### Modifications

Added the component in the src/components labeled as vertical_navbar.tsx. I also added it into the home page just to see if it looks like how it's supposed to. I didn't add any links to the icons though, because I realized the page for calendar was moved around and I wasn't sure what the clock referred to.
### Testing Considerations

Visually, it looks fine, but for the height of the navbar I wasn't sure how long it should be.

### Pull Request Checklist

- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [x] The developer name is specified
- [x] The summary is completed
- [x] Assign reviewers

### Screenshots/Screencast

Screenshot 2024-04-19 at 9 49 51 PM

### CODE REVIEW

File: src/app/components/vertical_navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { Home, CalendarDays, Clock }  from "lucide-react";
+import Link from "next/link";
+
+export default function Vertical_Navbar_Component() {
+  return (
+    <>
+    <ul className="m-0 h-max p-0 pb-96 w-10 bg-stone-400">
```
Comment (jinwu23) [inline src/app/components/vertical_navbar.tsx]: When I looked at the navbar, it did not expand to the bottom of the screen. I changed the padding to pb-full and the height to h-screen and it worked out

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import Navbar from "@components/Navbar";
+import Vertical_Navbar_Component from "@components/vertical_navbar";
 
 export default function Home() {
   return (
     <main>
       <h1>Volunteer Management System Home Page</h1>
+      <Vertical_Navbar_Component></Vertical_Navbar_Component>
```
Comment (jinwu23) [inline src/app/page.tsx]: For components that don't wrap any other ones, you can just do this tag <Vertical_Navbar_Component />

**Review by jinwu23 — APPROVED** [review]

File: src/app/components/vertical_navbar.tsx (hunk 1 above)
Comment (soramicha) [reply src/app/components/vertical_navbar.tsx]: Ah, sorry about that!

File: src/app/page.tsx (hunk 2 above)
Comment (soramicha) [reply src/app/page.tsx]: I understand!

**Pull request merged**